### ISSUE TITLE
Import error on index.d.ts

### ISSUE THREAD

**Issue by SYChoi** [nodevault/node-vault#241]
index.d.ts file has an error since request is not a dependency anymore.

**Comment by odubuc** [comment]
Agreed, same issue here, trying to install node-vault for the first time and it cannot compile.

> node_modules/node-vault/index.d.ts:7:26 - error TS2307: Cannot find module 'request' or its corresponding type declarations.
> 7 import * as request from "request";

**Comment by odubuc** [comment]
related to this PR ? https://github.com/nodevault/node-vault/pull/146
I added "@types/request" to my devDependencies to temporarly fix the issue..

**Comment by aviadhahami** [comment]
addressed in https://github.com/nodevault/node-vault/pull/257
